### DISCUSSION TITLE
Problem in loading .ex.dcm files

### DISCUSSION THREAD

**Discussion by Himanshu1331** [marksgraham/OCT-Converter#113] [General]
Hi

I have started working with Retinal OCT images, I have image files in **.ex.dcm** format, which I am not able to load using OCT- Converter. Kindly help me with this.

Regards

**Comment by marksgraham** [comment]
Hi there,

Would really need some information on what scanenr type this came from, but as a first pass you might have luck with the standard dicom reader:
https://github.com/marksgraham/OCT-Converter/blob/main/examples/demo_dicom_extraction.py

**Reply by Himanshu1331** [reply]
Thank you Marksgraham, 

It is not working .ex.dcm format
